### PR TITLE
style: make code shorter and clearer

### PR DESCRIPTION
**Pull request by HelKyle**

### CODE REVIEW

**Comment by emilkowalski** [comment]
This is awesome, can we not use react imports tho? Just a personal preference that I'd like to keep in this lib.

File: src/index.tsx [hunk 1]
```diff
@@ -135,7 +148,7 @@ function Root({
     return (window.innerWidth - 26) / window.innerWidth;
```
Comment (emilkowalski) [inline src/index.tsx:148]: We could get rid of this magic number as well, this is the top offset.

File: src/index.tsx [hunk 2]
```diff
@@ -474,66 +487,7 @@ function Root({
...
-
-  React.useEffect(() => {
-    // This is needed to force Safari toolbar to show **before** the drawer starts animating to prevent a gnarly shift from happenning
-    if (isOpen && isIOS()) {
-      setPositionFixed();
-    } else {
-      restorePositionSetting();
-    }
-  }, [isOpen]);
+  usePositionFixed(isOpen);
```
Comment (emilkowalski) [inline src/index.tsx:490]: Let's move it to the top of the function

Comment (HelKyle) [reply src/index.tsx]: Done

**Comment by emilkowalski** [comment]
Thank you so much, this is a great improvement.

**Pull request merged**